07-map/put.go

```diff
@@ -0,0 +1,17 @@
+package main
+
+import "fmt"
+
+func main() {
+	ages := map[string]int{
+		"Baby":   0,
+		"Justin": 45,
+		"Donald": 70,
+		"Canada": 150,
+	}
+	// START OMIT
+	ages["Baby"] = 1
+	ages["Baby2"] = 0
```

Comment:
Maybe a different name here? `Baby2:2` is a tad confusing to read vs. say `SecondChild:2`